iktomi/cli/base.py

```diff
@@ -77,8 +77,6 @@ def perform_auto_complete(commands):
                     k,v = splited
                 elif len(splited) == 1:
```

Comment:
For example, `test_function_as_command` have the case of `--opt` . 
For me it is not obvious that  `--opt=` is the same as `--opt`. For me `--opt=` is more similar to `--opt=""`